doc/ref_impl/cntt-ri/chapters/chapter04.md

```diff
@@ -7,56 +7,62 @@
 * [4.1 Introduction](#4.1)
 * [4.2 Lab Requirement](#4.2)
 * [4.3 Lab Topology](#4.3)
-* [4.4 Lab HW Spec](#4.4)
-* [4.5 Lab Use Guidelines](#4.5)
+* [4.4 Lab Use Guidelines](#4.4)
 
 
 <a name="4.1"></a>
-## 4.1 Introdcution
+## 4.1 Introduction
+This chapter provides the laboratory hardware requirements needed to deploy the reference implementation.  These requirements represent a minimum set of requirements, where different labs may have hardware or capabilities exceeding these minimum specifications.  
 
+The lab provides a set of bare-metal servers, supported by a dedicated network infrastructure.  Users should be provided access to the servers, their operating systems, and out-of-band management (i.e. IPMI, IDRAC, ILO, etc.).  
 
 <a name="4.2"></a>
 ## 4.2 Lab Requirement
 
-- Controller Nodes:
-  - 3 x
-    - 2x dual-port 10Gbps NIC.
-    - 2.2GHz 14C/28T.
-    - 256GB RAM.
-    - 10TB HDD.
-    - 3.2TB SSD.
-- Compute Node
-  - 10x
-    - 2x dual-port 25Gbps NIC.
-    - 2.2GHz 24C/48T.
-    - 512GB RAM.
-    - 2TB HDD
-    - Supports 2x Nodes for B/N/C flavors
-- Jump/Baremetal manager
-  - 1x
-    - 2x dual-port 10Gbps NIC.
-    - 2.2GHz 14C/28T.
-    - 256GB RAM.
-    - 10TB HDD
-- Networking
-  - 1x Spine Switch
-    - Total: 32x100G
-  - 2x Leaf Switch
-    - 48 x 25/10G
-    - 6 x 100G
-- 1x 48u Rack
-- Cables and Transcievers
+Labs are organized into one or more *Pods*, where the *Pod* provides the complete set of resources [servers and network(s)] necessary to support the reference implementation installation and operation.  
+
+A CNTT compliant *Pod* shall meet the following requirements.
+
+1. One (1) bare-metal server dedicated as a Jump / Test Host
+2. Six (6) bare-metal servers, serving as either compute or controllers
+3. A configured network topology allowing for: Out-of-Band Management, Admin, Public, Private, and Storage Networks
+
+The Jump Host / Test Host system will be utilized to install and manage the operation of the *Pod*.  For example, the system can be used to host the installer software used to deploy and configure OpenStack on the other bare-metal servers.  Once installed, the system may be used to run test automations or it may be used to generate test traffic intended to measure the performance of a VNF operating within the *Pod*.  To support these operations, the bare-metal server provided for the Jump / Test Host shall meet the same hardware requirements as compute / controller, as outline below.  In this context, the system may be considered as a miscellaneous use  or general purpose system.
+
+<a name="4.2.1"></a>
+### 4.2.1 Bare-metal Server Requirements
```

Comment:
There is a need to specify different categories of compute (NI, Basic, CI). each can have different specs. what was wrong with the original spec?

Reply:
There were not differences in the original specs for what was actually different (from a hardware perspective) for the 3 categories.  So, for release 1 or the ref. imp. documentation I'm proposing this simplification.  As the work matures, and we can clearly identify the differences (i.e. hardware needs), we can update these specifications for release 2, etc.  This allows labs to pace the role out of the hardware, etc.

Reply:
@rabi-abdel As a practical matter, IMHO, it is highly unlikely that enterprises in their private clouds will have different hardware for the different categories (B, NI, CI) -- except maybe for CI -- as it would lead to operational inefficiencies and will also partition the "cloud" by category.  Public clouds, on the other hand, have the necessary distribution of workload types that can justify the intended hardware by category variations.

Reply:
@rabi-abdel To add to the above, there were two relevant notes:

1) For various reasons, including it being part of the original UNH lab proposal, we've aligned that the lab, at least this lab, will be homogeneous, and therefore all Instance Types (IT) will run on the same server config/SKU, be it the concurrently on the same host (i.e. mixed IT types/host) or independently on a host dedicated to a particular IT.

2) There was a lot of discussion around the "jump" server (aka Jump Host (JH)), which is part of the POD, but outside of the NFVI, serving a supervisory role. The anchor driver for the JH is to host the VPN gateway to allow remote user access to the POD across the public internet, for control plane, OOB mgmt and tenant routeable internal and non-routeable networks (i.e. visibility to all networks in the POD). During the discussion, requirements were also raised for a location to host both instrumentation (e.g., Functest, iPerf, etc.) and instrumentation mgmt (e.g., Ixia client and apps of such ilk). The original concept was, when the JH was envisioned to merely host a VPN, why not make it a VM and put all PODs' VPN servers on the same host, rather than dedicating a 44 core, 2 socket server w/ 1/2 a TB of RAM, to perform a function a Raspberry Pi could handle, thus improving the total number of usable infra servers in the lab.
HTH, -Mark

Reply:
OK happy with the approach of homogeneous approach for now until the need to adjust.